datasets/us/fincen_msb/us_fincen_msb.yaml

```
@@ -0,0 +1,41 @@
+title: "MSB Registrant List"
+entry_point: crawler.py
+url: https://www.fincen.gov/msb-state-selector
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+coverage:
+  frequency: daily
+  start: 2024-09-10
+summary: >
+  The Money Services Business (MSB) Registrant List includes individuals and entities 
+  registered as MSBs operating in the United States.
+description: |
+  The MSB Registrant List is a directory of businesses and individuals engaged in money 
+  services within the United States, as mandated by the Financial Crimes Enforcement 
+  Network (FinCEN). These services include money transmitting businesses, currency dealers 
+  or exchangers, check cashers, issuers of traveler's checks, money orders, or stored value,
+  the US Postal Service, etc.
+prefix: us-fincen-msb
```

Looks like it can safely be trimmed

```suggestion
prefix: us-msb
```